contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,282 @@
+# Proposal to Standardize Development Process
+
+## Links
+```
+---
+Links:
+- Issues:
+  - [someIssueURL]()
+- PRs:
+  - https://github.com/kubernetes/community/pull/967
+- MailingListDiscussions:
+  - https://groups.google.com/forum/#!topic/kubernetes-dev/65A-3ULYPB0
+- Documentation:
+  - [someDocsLinkURL]()
+```
+
+## Responsible SIG(s)
+
+- SIG Release
+- SIG PM
+- SIG Architecture
+- SIG Governance
```

Comment:
We don't have a SIG Governance yet, though we may need one.

Also, you omitted SIG Testing here, though mentioned it in the PR.